test/Microsoft.AspNet.Mvc.WebApiCompatShimTest/ApiControllerTest.cs

```diff
@@ -419,7 +420,7 @@ public void ApiController_StatusCode()
             var result = controller.StatusCode(HttpStatusCode.ExpectationFailed);
```

Comment:
we have more than 400 uses of the `System.Net.HttpStatusCode` enum.  will those get replaced in a separate PR?